src/Campaigns/Blocks/CampaignCover/render.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Repositories\CampaignRepository;
+
+if ( ! isset($attributes['campaignId'])) {
+    return;
+}
+
+/** @var Campaign $campaign */
+$campaign = give(CampaignRepository::class)->getById($attributes['campaignId']);
+
+if ( ! $campaign) {
+    return;
+}
+
+$campaignMediaSetting = $campaign->image;
+
+$altText = $attributes['alt'] ?? 'Campaign cover image';
```

Comment:
Let's use the WP translate function here to make this string translatable.

Reply:
Thanks! 8790ed48eb39cb9ccf48ce285ea1e3e791e14802